lib/board.rb

```diff
@@ -0,0 +1,39 @@
+class Board
+  def initialize
+    @board = [
+      %w[1 2 3],
+      %w[4 5 6],
+      %w[7 8 9]
+    ]
+  end
+
+  def display_board
+    i = 0
+    while i < 3
+      x = 0
+      while x < 3
+        print @board[i][x]
+        print '|'
+        x += 1
+      end
+      i += 1
+      puts ''
+    end
```

Comment:
You shouldn't use any puts or print outside your `main` file. **Tip:** That doesn't mean you can't return a string and then puts it on your `main.rb `file :wink: